python/ray/experimental/data/dataset.py

```diff
@@ -858,18 +887,86 @@ def format_batch(batch: Block, format: str) -> BatchType:
         if batcher.has_any() and not drop_last:
             yield format_batch(batcher.next_batch(), batch_format)
 
-    def to_torch(self, **todo) -> "torch.utils.data.IterableDataset":
-        """Return a Torch data iterator over this dataset.
+    def to_torch(self,
+                 label_column: str,
+                 feature_columns: Optional[List[str]] = None,
+                 label_column_dtype: Optional[torch.dtype] = None,
+                 feature_column_dtypes: Optional[List[torch.dtype]] = None,
+                 prefetch_blocks=0):
+        """Return a Torch IterableDataset over this dataset.
+
+        Note that you probably want to call ``.split()`` on this dataset if
+        there are to be multiple Torch workers consuming the data.
+
+        Return a TF Dataset over this dataset.
+
+        Each element in IterableDataset will be a list consisting of 2
```

Comment:
tuple or list?

Reply:
I confirmed that it's a list. Seems like `DataLoader` converts it underneath the hood.